receiver/awscontainerinsightreceiver/receiver.go

```diff
@@ -344,13 +349,38 @@ func (acir *awsContainerInsightReceiver) initNVMeEbsScraper(ctx context.Context,
 		TelemetrySettings: acir.settings,
 		Consumer:          &decoConsumer,
 		Host:              host,
-		ScraperConfigs:    nvme.GetScraperConfig(hostInfo),
+		ScraperConfigs:    nvme.GetEbsScraperConfig(hostInfo),
 		HostInfoProvider:  hostInfo,
 		Logger:            acir.settings.Logger,
 	}
 
 	var err error
-	acir.nvmeScraper, err = prometheusscraper.NewSimplePrometheusScraper(scraperOpts)
+	acir.nvmeEBSScraper, err = prometheusscraper.NewSimplePrometheusScraper(scraperOpts)
+	return err
+}
+
+func (acir *awsContainerInsightReceiver) initNVMeLISScraper(ctx context.Context, host component.Host, hostInfo *hostinfo.Info, localNodeDecorator stores.Decorator) error {
+	decoConsumer := decoratorconsumer.DecorateConsumer{
+		ContainerOrchestrator: ci.EKS,
+		NextConsumer:          acir.nextConsumer,
+		MetricType:            ci.TypeNodeNVME,
```

Comment:
I thought we created a new node type for these EMF logs?

Reply:
I see this gets overwritten with a Type label set in the processor - https://github.com/aws/amazon-cloudwatch-agent/pull/1943.
But its unclear why these have to be out of sync. Not necessarily an issue with your changes, since i see its the same for the EBS as well - but something we should just fix for both in a follow up.